samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_eu.properties

```diff
@@ -335,3 +335,7 @@ download=Jaitsi
 obj=Helburuak
 keywords=Gako-hitzak
 objPercent=Ehuneko zuzena
+
+# S2U-15
+submission_nav_next_submission=Next submission
+submission_nav_previous_submission=Previous submission
```

Comment:
Hi @PabloSanRoman,
could you help with the following two translations please?
```
submission_nav_next_submission=Next submission
submission_nav_previous_submission=Previous submission
```
Greetings,
Markus

Reply:
Hi @stetsche 
of course, here it is: 

submission_nav_next_submission=Hurrengora
submission_nav_previous_submission=Aurrekora

Reply:
Thanks @PabloSanRoman !